src/topology/opens.lean

```diff
@@ -93,6 +93,13 @@ begin
 end
 /- Inf -/ _ rfl
 
+lemma le_def {U V : opens α} : U ≤ V ↔ (U : set α) ≤ (V : set α) :=
+by refl
+
+@[simp] lemma mk_inf_mk {U V : set α} {hU : is_open U} {hV : is_open V} :
+  (⟨U, hU⟩ ⊓ ⟨V, hV⟩ : opens α) = ⟨U ⊓ V, is_open_inter hU hV⟩ := rfl
+@[simp] lemma coe_inf {U V : opens α} : ((U ⊓ V : opens α) : set α) = (U : set α) ⊓ (V : set α) := rfl
```

Comment:
Are lemmas like this supposed to be tagged with `norm_cast` or is that just for numbers?

Reply:
The first one doesn't actually have a coercion, the type ascription is just there so lean knows what `⟨U, hU⟩` is meant to mean.

The second one, however, is a good `@[norm_cast]` lemma, so I've added that, thanks!